app/models/concerns/fiscal_year_mixin.rb

```diff
@@ -9,5 +11,11 @@ def self.which_fiscal_year(year, month)
         year
       end
     end
+
+    def self.for_fiscal_year(year)
```

Comment:
This was a `scope` and now a class method? Are they, strictly speaking, the same thing?

http://blog.plataformatec.com.br/2013/02/active-record-scopes-vs-class-methods/ was illuminating to me.